setup.py

```diff
@@ -39,7 +39,7 @@
     ],
     install_requires=[
         'msgpack', 'trio>0.8', 'async_generator', 'colorlog', 'wrapt',
-        'trio_typing',
+        'trio_typing', 'pdbpp',
```

Comment:
This is actually required as a dependency since we needed something where we could hook into the "tear down hooks" (i.e. where a user releases the debugger and let's code run again using either of the `continue` or `quit` commands).